x/auth/tx/direct_aux.go

```diff
@@ -52,17 +52,14 @@ func (signModeDirectAuxHandler) GetSignBytes(
 		return nil, errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "got empty address in %s handler", signingtypes.SignMode_SIGN_MODE_DIRECT_AUX)
 	}
 
-	feePayer := protoTx.FeePayer()
+	payer := protoTx.FeePayer()
+
+	feepayer := sdk.AccAddress(payer)
```

Comment:
nit: for simplicity, we could just do `sdk.AccAddress(payer).String()` below and revert the variable change above.